calculadora_do_cidadao/adapters/cpi.py

```diff
@@ -0,0 +1,40 @@
+from datetime import date
+from decimal import Decimal
+from typing import NamedTuple
+from urllib.parse import urlencode
+
+from rows.fields import DecimalField, DateField
```

Comment:
Can we keep alphabetical order?

```suggestion
from rows.fields import DateField, DecimalField
```